### PR TITLE
Fix print_wrapped to properly parse "\x5" newline character

### PR DESCRIPTION
**Pull request by Choms**
:rainbow: Hi there!

While using Thor I found that the print_wrapped function behaves weirdly when manually inserting \x5 sequence (as per your docs) to force a line break. The original function doesn't takes those line breaks into consideration when calculating the length of the line, and the result is some extra weird line breaks on a long paragraph that includes such sequences (e.g. giving several execution examples).

I've rewrote this function, I reckon it's less "stylish", ~~and has a side effect that is unlikely a print_wrapped will use the whole terminal size~~, but it won't break intended multi-line paragraphs either.

Long story short, I'm breaking a paragraph in words, doing the "\x5" to "\n" replacement and keeping a counter that tells me when the line is over the terminal size, so it places a line break.

This change doesn't interfere with core functionality and I've tested it, feel free to review it and if you see fit, merge (I've also increased the version number).

PS: Both in your "master" branch and my fork I'm getting 4 errors related to Readline spec when running `bundle exec thor spec` locally ¯\\\_(ツ)_/¯

Cheers!

### CODE REVIEW

**Comment by Choms** [comment]
mmmm I see travis doesn't likes the changes, I'll look into it when I have some time (over the weekend likely) or if anyone else has a hint (it must be related to travis terminal_width - maybe changing the spec to expect an extra newline would do the trick) help is appreciated :)

**Comment by Choms** [comment]
I did some weird calculation with the indentation (for some reason I didn't saw the indent being subtracted already from the terminal width).
I fixed that on the last commit, now it's all passing and actually using the whole terminal width, should be good to merge.

**Comment by MaxLap** [comment]
I can't speak about the details of the change, but the version shouldn't be changed like that as part of the commit. When the maintainer does a release, he will increment it.

**Comment by rafaelfranca** [comment]
You removed the version file instead of only removing the version change. This is why the tests are failing.

**Comment by Choms** [comment]
yea I see feel free to add it merge or just close the PR, I thought it
would be useful, I really don't care about versioning discussions

**Comment by rafaelfranca** [comment]
Sounds good. Can you share at least which string you are using to test so I can write a test to make sure this behavior keep fixed.

**Comment by rafaelfranca** [comment]
Awesome! Thank you so much. I'll merge and add the tests myself.
